### DISCUSSION TITLE
How to get count of total records

### DISCUSSION THREAD

**Discussion by jbardnz** [netsuitephp/netsuite-php#105] [Q&A]
Hi there,

I was wondering if anyone has a code example of how to get a count of total records. E.g. I am trying to get the number of contacts modified in the last month.

Thanks!

**Comment by jrebs** [comment]
Does the response to your search have a `totalRecords` property?

**Reply by jbardnz** [reply]
Yes at the moment I am doing a normal search, limiting it to two items per page and getting the totalRecords. 

I wasn't sure if NetSuite had a way to get the total records without doing a full search.
